### ISSUE TITLE
Polymode support

### ISSUE THREAD

**Issue by Pitometsu** [millejoh/emacs-ipython-notebook#171]
It's feature request: use [polymode](https://github.com/vspinu/polymode) to get most adequate [markdown](http://jblevins.org/projects/markdown-mode/) support.

**Comment by millejoh** [comment]
I looked at this a while back, unfortunately polymodes method of using indirect buffer corrupts the ein's data structures for representing the notebook in a way I'm not quite competent enough to resolve. I do know once upon a time nxhtml had a multimode that worked with ein, but that code no longer works with recent versions of emacs.

What I have been able to do is implement editing similar to what org-mode has for source blocks. If you call `C-'` on the cell it will open a dedicated buffer with the (I hope) appropriate major-mode enabled, i.e. python or markdown. Hope that is at least a viable workaround.
